src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+}
```

Comment:
The use of a universal selector (`*`) for resetting margins is generally not recommended as it can lead to performance issues. Consider using a more specific selector or a CSS reset/normalize file.